docker/docker-compose.yaml

```diff
@@ -0,0 +1,47 @@
+version: '3'
+
+networks:
+  rolelive-net:
+    driver: bridge
+
+services:
+
+# Python
+  python:
+    image: 'oriolfilter/python-rolelive:first_try'
+    volumes:
+      - ./../rolelive2.py:/code/app.py
+#    env_file:
+#      - ./rolelive.env # rolelive v1
+    restart: always
+    networks:
+      - rolelive-net
+# Memcached
+  memcached:
+    image: 'bitnami/memcached:latest'
+    environment:
+      MEMCACHED_MAX_CONNECTIONS: "${MEMCACHED_MAX_CONNECTIONS}"
+      MEMCACHED_THREADS: "${MEMCACHED_THREADS}"
+      MEMCACHED_USERNAME: "${MEMCACHED_USERNAME}"
+      MEMCACHED_PASSWORD: "${MEMCACHED_PASSWORD}"
+#      - MEMCACHED_PASSWORD_FILE=file_with_admin_passwd
+#    command: /opt/bitnami/scripts/memcached/run.sh -vvv
+#    volumes:
+#    - ./memcached.conf:/opt/bitnami/memcached/sasl2/memcached.conf
```

Comment:
Commented code is a bad practice. Version control and documentation exists for a reason. If the commented lines should exist, uncomment them. If the commented lines no longer apply, remove them. If the commented lines may be required for a certain purpose, comment in the ``README`` how they should be used.